src/pages/components/SecondaryBar.qml

```diff
@@ -100,8 +100,14 @@ Column {
         Browser.IconButton {
             width: midIconWidth
             icon.source: "image://theme/icon-m-file-download-as-pdf"
-            active: webView.contentItem && webView.contentItem.active && !webView.loading && !DownloadManager.pdfPrinting
-            onTapped: savePageAsPDF()
+            active: webView.contentItem && webView.contentItem.active && !webView.loading
+            onTapped: {
+                if (DownloadManager.pdfPrinting) {
+                    pdfPrintingNotice.show()
+                } else {
+                    savePageAsPDF()
```

Comment:
Maybe you can also show a Notice when the printing has been started.

Reply:
Printing to pdf works through download manager. Transfers notifies when printing starts and finishes/fails.